charts/sn-platform/templates/proxy/proxy-cluster.yaml

```diff
@@ -149,7 +149,7 @@ spec:
       forwardAuthorizationCredentials: "true"
       authenticationProviders: {{ include "pulsar.authenticationProviders" . }}
 {{- include "pulsar.authConfiguration" . | indent 6 }}
-      superUserRoles: "{{ .Values.auth.superUsers.broker }}"
+      superUserRoles: "{{ .Values.auth.superUsers.proxy }}"
```

Comment:
when the `proxyRolesEnabled` is false, also use `.Values.auth.superUsers.proxy`?

Reply:
I think it should have been a mistake before